nav2_costmap_2d/README.md

```diff
@@ -0,0 +1,21 @@
+# Nav2 Costmap_2d
+
+The costmap_2d package is responsible for building a 2D costmap of the environment, taking observations from sensors and 
```

Comment:
This first sentence is a little confusing. Maybe try; "The costmap_2d package is responsible for building a 2D costmap of the environment, consisting of several "layers" of data about the environment. It can be initialized via the map server or a rolling window (what does that mean?) and updates the layers by taking observations from sensors"